eventuate-log-cassandra/src/it/scala/com/rbmhtechnology/eventuate/LocationSpecCassandra.scala

```diff
@@ -98,6 +97,39 @@ object SingleLocationSpecCassandra {
   }
 }
 
+trait SingleLocationSpecNoIndexingCassandra extends SingleLocationSpec with LocationCleanupCassandra {
```

Comment:
This is unnecessary code duplication with `SingleLocationSpecCassandra`. Instead, try implementing `EventLogSpecCassandraNoIndexing` as

```scala
class EventLogSpecCassandraNoIndexing extends TestKit(ActorSystem("test", EventLogSpecCassandra.config)) with EventLogSpec with SingleLocationSpecCassandra {
  import SingleLocationSpecCassandra._

  override def logProps(logId: String, failureSpec: TestFailureSpec, indexProbe: ActorRef): Props =
    TestEventLog.props(logId, failureSpec, indexProbe, batching, aggregateIndexing = false)
}
```

and remove the `SingleLocationSpecNoIndexingCassandra` trait.